engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala

```diff
@@ -83,6 +84,9 @@ class K8sDeploymentManager(modelData: BaseModelData, config: K8sDeploymentManage
                       savepointPath: Option[String]): Future[Option[ExternalDeploymentId]] = {
     val scalingOptions = determineScalingOptions(canonicalProcess)
     for {
+      resourceQuotas <- k8s.list[ResourceQuotaList]()
+      oldDeployment <- k8s.getOption[Deployment](objectNameForScenario(processVersion, config.nussknackerInstanceName, None))
+      _ <- K8sPodsResourceQuotaChecker.hasReachedQuotaLimit(oldDeployment.flatMap(_.spec.get.replicas), resourceQuotas, scalingOptions.replicasCount)
```

Comment:
Why result of invoking this method is ignored?

Reply:
1. its just validation 2. done